app/src/main/res/layout/activity_main.xml

```diff
@@ -1,29 +1,70 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
-    android:id="@+id/activity_main"
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:gravity="center"
+    android:gravity="center_horizontal"
     android:orientation="vertical">
-
     <TextView
-        android:id="@+id/count_label"
+        android:id="@+id/text_expression"
```

Comment:
In general, if you are going to use a view with Kotlin and synthetic imports, please use camelCase variable notation.
```suggestion
        android:id="@+id/textExpression"
```